cards.ini

```diff
@@ -49,3 +49,51 @@ j how = Jackson Howard
 our lord and savior = Jackson Howard
 pizza field = Giordano Memorial Field
 nre = Net-Ready Eyes
+rpc = Red Planet Couriers
+piggy bank = Bankroll
+flowers for = Algernon
+hail = Hydra
+free willy = Darwin
+ttw = The Turning Wheel
+sauron = Hunter
+darude = Sand Storm
+hoot = Owl
+piot = Peace in Our Time
+nice car = Sports Hopper
+sweet car = Advanced Concept Hopper
+folding at home = Data Folding
+gun = Unregistered S&W '35
+pistol = Unregistered S&W '35
+catch me if you can = Gingerbread
+twitch.tv = Daily Casts
+hart = Harmony AR Therapy
+tca = The Class Act
+overwatch = Loot Box
+netflix = Tiered Subscription
+wow = Daily Quest
+boxer girl = Leela Patel: Trained Pragmatist
+proco = Professional Contacts
+mother brain = Ikawah Project
+hpt = High-Profile Target
+yogurt = Economic Warfare
+rebecca = Friday Chip
+doof = Diversion of Funds
+fugu = Puffer
+knock knock = No One Home
+just in time = Compile
+godzilla = Gbahali
+mopus = Magnum Opus
+saccon = Sacrificial Construct
+tb = Tollbooth
+petty = Punitive Counterstrike
+sft = Successful Field Test
```

Comment:
Going to ask to strike this out - this commonly also refers to Shipment from Tennin as well.

Reply:
Heard the commentators refer to this to SFT during stream, but makes sense. Thanks!